### ISSUE TITLE
How to test without GT(EDVR)

### ISSUE THREAD

**Issue by nobu241** [xinntao/EDVR#179]
Please tell me how to test without GT

I would like to try it with an actual frame without GT

**Comment by fadwaSAA** [comment]
I have the same question
 can you please tell us how to test EDVR pre-trained models on custom datasets without ground truth(GT)??

**Comment by zbrnwpu** [comment]
Has your problem been solved?

**Comment by zbrnwpu** [comment]
> I have the same question
> can you please tell us how to test EDVR pre-trained models on custom datasets without ground truth(GT)??

Has your problem been solved?

**Comment by sisrfeng** [comment]
+1

**Comment by noobtoob4lyfe** [comment]
Yes, this is very confusing.  Please provide an EDVR colab if possible.  Thanks!
